docs/upgrading.md

```diff
@@ -74,6 +23,8 @@ You first need to select from and to which version you wish to upgrade, by defau
 
 💡 Major updates will show an "useful content" section on the top with links to help you out when upgrading.
 
+> Or you can run the `npx react-native upgrade`, which will automatically check your current version and the latest version available and will show you the link to the Upgrade Helper page with the versions already selected.
+
 #### 2. Upgrade dependencies
 
 The first file that is shown is the `package.json`, it's good to update the dependencies that are showing in there. For example, if `react-native` and `react` appears as changes then you can install it in your project by running `yarn add`:
```

Comment:
Again not related to this PR.
Could you please also add the `npm` equivalent (`npm install`) for `yarn add` throughout this doc?
e.g.: L34, L35

You should utilize the TabItems and Tab components, just search for TabItem in vscode/in your editor :)

Reply:
Sure, let me create PR :)